equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/Ae.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+
+public class Ae  {
```

Comment:
I would suggest a better name.  AE means nothing and everything. It is also missing the Model suffix